Druid/DRCogitStackToRegisterMappingGenerator.class.st

```diff
@@ -68,6 +68,13 @@ DRCogitStackToRegisterMappingGenerator >> returnNodeForCompiledMethod [
 	^ RBLiteralNode value: 0
 ]
 
+{ #category : #visiting }
+DRCogitStackToRegisterMappingGenerator >> rtlExpressionForValue: aValue [
+
+	variables at: aValue result ifPresent: [ :var | ^ RBVariableNode named: var ].
+	^ super rtlExpressionForValue: aValue
+]
+
```

Comment:
Why sometimes the order of the methods change? Could be a _non-deterministic situation_ in.... Fuel/Iceberg?

Reply:
tonel